__tests__/app.test.ts

```diff
@@ -145,6 +145,27 @@ describe('🧪 Express Application', () => {
 					.expect(400);
 			});
 		});
+
+		describe('GET /api/users', () => {
+			it('200: should get all users', () => {
+				return request(app)
+					.get('/api/users')
+					.expect(200)
+					.then(({ body: { success, data, count } }) => {
+						expect(success).toBe(true);
+						expect(count).toBe(2);
```

Comment:
3?

Reply:
3?

Reply:
![image](https://github.com/user-attachments/assets/2721acd3-563a-44cd-bf7d-5791af49ee9c)

Reply:
<img width="340" alt="image" src="https://github.com/user-attachments/assets/4d22deaa-aafb-4eb9-9cd5-f237ea96025d">

Reply:
and what does the beforeAll function look like

Reply:
How weird. It should be something completely different

Reply:
https://github.com/clowder-cloud/backend/blob/dc9976101f3a1ddba6a74c49261c0c1e0f804da7/__tests__/app.test.ts#L8

Reply:
Look at the same code in the main branch. Seems the PR for that got pulled in but it didnt change that. This "version control" lark annoys me.

Reply:
Presumably you made this branch from main before that PR got merged into main

Reply:
Are you using work trees?

Reply:
Never heard of them before.

Reply:
Looks like you didn't branch from main when you started?
![image](https://github.com/user-attachments/assets/42d23c3c-9876-4f4e-b019-79edf0642b65)

Reply:
Ignore me, I'm reading that wrong - it doesn't show main like I expected it to, but it looks like you did branch from it properly.